src/runtime_src/core/edge/ps_kernels/xrt/tests/validate/ps_bandwidth_test/ps_bandwidth.cpp

```diff
@@ -9,6 +9,14 @@
 
 #include "xrt/xrt_kernel.h"
 
+constexpr unsigned long long int operator "" _Kb(unsigned long long int n) {
+  return n * 1024;
+}
+
+constexpr unsigned long long int operator "" _Mb(unsigned long long int n) {
+  return n * 1024 * 1024;
+}
```

Comment:
What a great idea. :-)
But if we go that path, there is nothing about bit quantity here and you should follow the standard naming: 
https://en.wikipedia.org/wiki/Binary_prefix#kibi
https://en.wikipedia.org/wiki/Binary_prefix#mebi

Reply:
While you are on it, to have your colleagues benefiting from it, what about having this it in a private XRT header which is only included by the `.cpp` needing this?

Reply:
This is a source executable that does not have access to the majority of the XRT codebase, only a few include files for xrt_* types. If this was inside XRT proper we already have something like this.